batterybar/README.md

```diff
@@ -0,0 +1,31 @@
+<center>i3blocks battery bar</center>
+						
+	
+What does it do ?
+-----------------
+It displays the battery level in a set of five unicode squares (U+25A0).<br>
+It also changes color for more accuracy and to distinguish between<br>
+charging, discharging, charged and AC statuses.<br>
+	
+Left-clicking briefly shows the battery level in percent.<br>
+
+Installation
+------------
+1-Copy the battery_bar script into /usr/share/i3blocks/.<br>
```

Comment:
Please review the [markdown syntax](https://daringfireball.net/projects/markdown/syntax#list). Ordered lists are formatted with dots:

    1. foo
    2. bar